packages/api/endpoints/distribution.js

```diff
@@ -83,32 +84,32 @@ async function handleRedirectRequest(req, res) {
     username: getAccessTokenResponse.username
   });
 
-  return res.cookie(
-    'accessToken',
-    getAccessTokenResponse.accessToken,
-    {
-      expires: new Date(getAccessTokenResponse.expirationTime),
-      httpOnly: true,
-      secure: true
-    }
-  )
+  return res
+    .cookie(
+      'accessToken',
+      getAccessTokenResponse.accessToken,
+      {
+        expires: new Date(getAccessTokenResponse.expirationTime),
+        httpOnly: true,
+        secure: true
+      }
+    )
     .set({ Location: urljoin(distributionUrl, state) })
     .status(307)
     .send('Redirecting');
 }
 
 /**
- * Responds to a file request
+ * Ensure request is authorized through EarthdataLogin or redirect to become so.
  *
- * @param {Object} req - express request object
- * @param {Object} res - express response object
- * @returns {Promise<Object>} the promise of express response object
+ * @param {any} req
+ * @param {any} res
+ * @param {any} state
  */
-async function handleFileRequest(req, res) {
+async function ensureAuthorizedOrRedirect(req, res, next) {
```

Comment:
Is this authorized or authenticated?  I feel like it could be either.

Reply:
You have to authenticate to URS in order to be authorized, but what we're actually checking via the cookie is whether you are authorized, imo